src/utils/utils.ts

```diff
@@ -37,7 +37,6 @@ export const imageURL = (
 export const getNumItems = async (uuid: string) => {
```

Comment:
@EdwinGuzman @7emansell wondering how to refactor this to use the getAPIUri method, main concern is that getAPIUri returns the api response and getNumItems returns the numItems.